### PR TITLE
Add XmlNodeName attribute for export name matching

### PR DESCRIPTION
**Pull request by lwschan**
TimeLog.ReportingApi.Exporter compares XML node name against the property name in TimeLog.ReportingApi.SDK to get the right value for the export columns.
We have some columns implemented with different property name from the XML. So XmlNodeName attribute is created.
This attribute is used when the XML node name is different from the property name, to facilitate the export name matching.
Due to backwards compatibility issue, renaming the property is not a valid solution

### CODE REVIEW

File: TimeLog.ReportingApi.SDK/Project.cs [hunk 1]
```diff
@@ -262,6 +262,7 @@ public Project(XmlNode node, XmlNamespaceManager namespaceManager)
         /// <summary>
         /// Gets or sets the related project number
         /// </summary>
+        [XmlNodeName("No")]
         public string ProjectNo { get; set; }
```
Comment (oexenhave) [inline TimeLog.ReportingApi.SDK/Project.cs]: Why not just change the property name instead?

**Review by oexenhave — CHANGES_REQUESTED** [review]

File: TimeLog.ReportingApi.SDK/Project.cs (hunk 1 above)
Comment (lwschan) [reply TimeLog.ReportingApi.SDK/Project.cs]: Do we need to consider backwards compatibility? If someone has already built an app using ProjectNo, they would have to resolve the breaking changes in the next update.

Comment (oexenhave) [reply TimeLog.ReportingApi.SDK/Project.cs]: I agree. However, I propose we consider backwards compatibility in the API over the SDK. Unfortunately, this bug has been there for a while I guess. I rather want the SDK to be one-to-one with the API. What's your take?

Comment (lwschan) [reply TimeLog.ReportingApi.SDK/Project.cs]: I agree, I believe standardizing the names in SDK and API is preferable. It is a straight forward rule that is easy to apply. 
I have reverted the previous code change and updated the pull request, please review again, thanks!

**Review by oexenhave — APPROVED** [review]

**Pull request merged**